### ISSUE TITLE
Attacks producing only 0s

### ISSUE THREAD

**Issue by lordsyren** [Bathtor/EPSheet#79]
Sometimes when we click on a weapon to attack with it to produce a roll, the result is always zero.

I was able to figure out the issue. The specialization value sometimes just does not populate, which makes the formula #+#+#+#++((0-(0+0)+0)+0+0) instead of what it should be #+#+#+#+#+((0-(0+0)+0)+0+0). Checking/unchecking the specialization box fixes the issue.

**Comment by Bathtor** [comment]
Yeah, it's a known issue. I haven't been able to track it to anything I did on my side, so I'm pretending it's a Roll20 bug (of which there are many, so it's not unlikely to be true ;). As you mentioned there's an easy workaround, so it's not a high priority imo to track this down further.

I'll leave this issue open for now, so other people can see that it's a known issue (you are not the first person to ask about it, I think the last one wrote an email, though, so it didn't end up being documented here).

**Comment by lordsyren** [comment]
Thanks for responding so quickly.

Maybe this isn't the correct place to put this, but I just wanted to thank
you for the sheet. It's really helpful and it's removed so many headaches I
had about making my own sheet like I often do. Your video about how to put
things together was very nice, too!

**Comment by Bathtor** [comment]
> Maybe this isn't the correct place to put this, but I just wanted to thank you for the sheet. It's really helpful and it's removed so many headaches I had about making my own sheet like I often do. Your video about how to put things together was very nice, too!

Thank, I appreciate that :) Glad it helps.
